server/server_http.go

```diff
@@ -39,7 +39,12 @@ func (s *Server) webHandle(w http.ResponseWriter, r *http.Request) {
 	}
 	//api call
 	if strings.HasPrefix(r.URL.Path, "/api/") {
-		w.Header().Set("Access-Control-Allow-Headers", "authorization")
+		origin := r.Header.Get("Origin")
```

Comment:
CORS only affects on requests made from the browser, hence it should only be applied to ```webHandle``` not ```restAPIhandle```.